src/Microsoft.DotNet.Tools.Pack/Program.cs

```diff
@@ -48,6 +48,17 @@ public static int Main(string[] args)
                     path = Directory.GetCurrentDirectory();
                 }
 
+                if(!path.EndsWith(Project.FileName))
```

Comment:
if only there was some tool I could run post-commit to reformat, say a `dotnet fmt` tool ;P